src/scripts/components/App.js

```diff
@@ -11,8 +11,7 @@ import PlayContainer from 'containers/PlayContainer'
 import DebugContainer from 'containers/DebugContainer'
 import VideoManager from 'containers/VideoManagerContainer'
 
-import type { RouteMatch } from 'types/ApplicationTypes'
-import { setupKeystore } from 'utils/ParatiiLib'
+import type { Match } from 'react-router-dom'
```

Comment:
I realized that `react-router-dom` already had a better typedef for this property than we had hacked in